autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -252,7 +315,25 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             set_best_to_refit_full : bool, default = False
                 If True, will set Trainer.best_model = Trainer.full_model_dict[Trainer.best_model]
                 This will change the default model that Predictor uses for prediction when model is not specified to the refit_full version of the model that previously exhibited the highest validation score.
-                Only valid if `refit_full=True`.
+                Only valid if `refit_full` is set.
+            save_bagged_folds : bool, default = True
+                If True, bagged models will save their fold models. This enables bagged models to be capable of inference after `fit()`.
+                If False, bagged models will not save their fold models. This means that bagged models will not be valid models during inference.
+                    This should only be set to False when planning to call `predictor.refit_full()` or when `refit_full` is set and `set_best_to_refit_full=True`.
+                    Particularly useful if disk usage is a concern. By not saving the fold models, bagged models will take virtually no disk space during training.
+                    In a typical training run, this will reduce peak disk usage by >10x.
+                This parameter has no effect if bagging is disabled.
+            keep_only_best : bool, default = False
+                If True, only the best model and its ancestor models are saved in the outputted `predictor`. All other models are deleted.
```

Comment:
Add: 
"Use this if you only care about deploying the most accurate predictor with the smallest file-size and no longer need any of the other models AutoGluon trained during `fit()`."

Reply:
Added